test/jdk/java/util/zip/EntryCount64k.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2013 Google Inc. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is this OK? The header reads `DO NOT ALTER`.

Reply:
Yes, I think you're right. I based my decision off https://github.com/openjdk/jdk/pull/15454

But after reading the copyright guidelines, we do not modify third party copyright. It should probably be removed in that test as well.